Sources/Exporters/OpenTelemetryProtocol/trace/utils/HTTPClient.swift

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import Foundation
+
+/// Client for sending requests over HTTP.
+internal final class HTTPClient {
+    private let session: URLSession
+
+    convenience init() {
+        let configuration: URLSessionConfiguration = .ephemeral
```

Comment:
Is it worth specifying "wait for connectivity" or a session task priority (ensuring that OTLP data doesn't slow down UI updates)?

Reply:
Yes is a good idea, anyway we should also provide a constructor which accepts an existing URLSession to use